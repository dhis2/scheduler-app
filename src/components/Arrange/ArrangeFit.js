import React from 'react'
import { node } from '@dhis2/prop-types'
import styles from './ArrangeFit.module.css'

/**
 * Shrink container to fit contents
 */

const ArrangeFit = ({ children }) => {
    return <div className={styles.fit}>{children}</div>
}

ArrangeFit.propTypes = {
    children: node.isRequired,
}

export default ArrangeFit
