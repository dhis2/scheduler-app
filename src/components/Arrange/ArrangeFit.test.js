import React from 'react'
import { shallow } from 'enzyme'
import ArrangeFit from './ArrangeFit'

describe('<ArrangeFit>', () => {
    it('renders correctly', () => {
        const wrapper = shallow(<ArrangeFit>Text</ArrangeFit>)

        expect(wrapper).toMatchSnapshot()
    })
})
