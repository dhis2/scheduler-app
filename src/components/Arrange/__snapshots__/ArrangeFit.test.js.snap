// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ArrangeFit> renders correctly 1`] = `
<div
  className="fit"
>
  Text
</div>
`;
